api/STYLE.md

```diff
@@ -189,7 +187,7 @@ metadata. We describe these annotations below by category.
 * `option (udpa.annotations.file_migrate).move_to_package = "<package name>";`
   to denote that in the next major version of the API, the file will be moved to
   the given package. This is consumed by `protoxform`.
-* `option (udpa.annotations.file_status).work_in_progress = true;` to denote a
+* `option (xds.annotations.v3.file_status).work_in_progress = true;` to denote a
```

Comment:
@htuch any preference between `xds.annotations...` and `udpa.annotations...`? (not only for WIP but also other annotations)

Reply:
We're in the process of moving all `udpa.*` to `xds.*`, so the former is better.